docs/developer-docs/backend/choosing-language.md

```diff
@@ -3,35 +3,41 @@ sidebar_position: 1
 ---
 # Choosing a programming language
 
-To create canisters, it is typical to use a so-called CDK, a Canister Development Kit.
-The CDK's main tasks are to expose the public interface of a canister, manage memory, allow other canisters to be called, and to interface with the system API.
-Because the Internet Computer supports dapps compiled to standard WebAssembly modules, you can use many different programming languages to create your canisters.
-The two officially supported languages of the Internet Computer are [Motoko](/motoko/main/motoko.md) and [Rust](./rust/index.md).
-Motoko is specifically designed to support the unique features of the Internet Computer and to provide a familiar yet robust programming environment.
-Rust is a powerful and modern type-sound programming language with an active developer community.
-Of course, developers are not required to work in one (or both) of those languages. There are many different community-supported languages available as well, as you can see below in the list of available languages.
+To create smart contract canisters, it is typical to use a so-called CDK, a Canister Development Kit.
+The CDK's main tasks are to expose the public interface of a canister, manage memory, allow other canisters to be called, and to interface with the system API. Because the Internet Computer supports dapps compiled to standard WebAssembly modules, you can use many different programming languages to create your canisters.
 
-It is also possible to split your work between multiple languages. Different canister smart contracts talk to each other using the [Candid](./candid/index.md) language. What language works behind the candid interface, however, does not matter.
+In theory, any language that can be compiled into a WebAssembly module, can produce modules [tailored for the IC](../../references/ic-interface-spec.md) depoyable as ICP smart contract.
```

Comment:
I like this better; it prevents a potential footgun.

One tiny grammar nit with suggested change:
```suggestion
In theory, any language that can be compiled into a WebAssembly module, can produce modules [tailored for the IC](../../references/ic-interface-spec.md) depoyable as an ICP smart contract.
```